docs/AKSManualDeploymentGuide.md

```diff
@@ -7,16 +7,23 @@ This document explains how to create an AKS cluster manually for Scalar DL.
 Please follow [Create an AKS cluster](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal#create-an-aks-cluster) with the following Scalar DL specific settings.
 
 ### Scalar DL specific settings
-Please select the region and the resource group appropriately for your system.
-Also, please specify the same node pool name for `ledger` and `envoy`. It will help you to deploy them in the same node pool. The current custom config specifies `scalardlpool` for that, so please specify it if you don't have any preference.
-Accordingly, the node size for `ledger` and `envoy` should be selected appropriately. The recommended node size is `Standard D2s v3` if you don't have any preference.
+Select the appropriate region and resource group for Scalar DL environment creation. Create 2 subnets with prefix of at least `/22` for AKS, one subnet must be created with the name `k8s_ingress` to create an envoy load balancer.
+If you use a different subnet for the envoy load balancer it must be specified in the custom config file.
+Do not forget to specify the same node pool name for `ledger` and `envoy`. The current custom config uses `scalardlpool` for that, you can modify it based on your preferences.
+The node size for `ledger` and `envoy` should be selected appropriately. The recommended node size is `Standard D2s v3`.
 
-On the **Networking** page, configure the Kubernetes cluster in the same virtual network as the other resources, so please choose `Azure CNI` as **network configuration**.
-Kubernetes cluster deployment requires 388 addresses so please create a subnet accordingly.
+On the **Networking** page, configure the Kubernetes cluster in the same virtual network as the other resources: 
+ - Choose `Azure CNI` as **network configuration** and select the subnet other than `k8s_ingress`.
 
 ## Access AKS Cluster
 
 Prepare kube config file
 ```
 $ az aks get-credentials --resource-group <RESOURCE_GROUP_NAME> --name <AKS_CLUSTER_NAME>
 ```
+
+## Next steps
```

Comment:
First of all, is this doc redundant with `Manual deployment guide ScalarDL on Azure`?

Questions:
* Do we need this?
* Are you expecting users to read this and go to `Manual deployment guide ScalarDL on Azure` guide? If so, isn't it weird as an organization?
* If uses read `Manual deployment guide ScalarDL on Azure`, then the users can properly set the above things? (they don't seem to be documented in `Manual deployment guide ScalarDL on Azure`)

Reply:
@feeblefakie san,

Yes, It is redundant.
We can remove this documentation because we have a detailed document for  `Manual deployment guide ScalarDL on Azure` and `Manual deployment guide ScalarDL on EKS`

Reply:
Can I remove these documents?